docs/en/Tiered-Storage-on-Alluxio.md

```diff
@@ -1,88 +1,117 @@
 ---
 layout: global
-title: Tiered Storage on Alluxio
-nickname: Tiered Storage
+title: Alluxio Storage
```

Comment:
do we need to change the doc filename too, as it will show as a part of the Http URL and can be confusing?

Reply:
Updated the file name and existing links.